front-end/src/axios.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+const instance = axios.create({
+  baseURL: "https://developer-verse.herokuapp.com/",
+});
```

Comment:
this is good, but you'll want to change the base instance depending on your env

so if its in production that's the way to go, but when I'M developing I wanna be using `localhost`

Reply:
this is the deployed heroku app, are you saying i should be using my local backend database?

Reply:
The source code should run with a different base URL depending on if it's running locally on a dev's machine or if it's running in the test environment. So you'll be implementing environment variables for the React app in this case.

Reply:
it'll look like 

```
instance;

if (envvariable === "development) {
  instance = axios.create({baseURL: 'local})
} else {
  instance = axios.create({baseURL: 'deployed})
}